src/dynarec/rv64/dynarec_rv64_helper.h

```diff
@@ -501,6 +501,24 @@
         addr = geted(dyn, addr, ninst, nextop, &wback, a, x3, &fixedaddress, rex, NULL, I12, D); \
     }
 
+
+#define GETGY()                                 \
+    gd = ((nextop & 0x38) >> 3) + (rex.r << 3); \
+    gback = xEmu;                               \
+    gdoffset = offsetof(x64emu_t, ymm[gd])
+
+#define GETEY(a, D, I12)                                                                         \
+    if (MODREG) {                                                                                \
+        ed = (nextop & 7) + (rex.b << 3);                                                        \
+        fixedaddress = offsetof(x64emu_t, ymm[ed]);                                              \
+        wback = xEmu;                                                                            \
+    } else {                                                                                     \
+        SMREAD();                                                                                \
+        ed = 16;                                                                                 \
+        addr = geted(dyn, addr, ninst, nextop, &wback, a, x3, &fixedaddress, rex, NULL, I12, D); \
+        fixedaddress+=16;                                                                        \
```

Comment:
I don't think we can do `fixedaddress += 16` here, the value will overflow.

Reply:
> I don't think we can do `fixedaddress += 16` here, the value will overflow.

When fetching 256 bits of data, it needs to be offset by 128 bits

Reply:
> I don't think we can do `fixedaddress += 16` here, the value will overflow.

It won't overflow, I did test it with the demo and the result is correct

Reply:
please make sure you understand what the i12 is for.